### PR TITLE
[Hub Generated] Review request for Microsoft.CostManagement to add version stable/2019-01-01

### PR DESCRIPTION
**Pull request by bgsky**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/app/branch/bgsky/azure-rest-api-specs/dev-cost-management-Microsoft.CostManagement-2019-01-01...master).

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by bgsky** [comment]
Some validations failed, complaining about scope parameter not available in examples in Json files.

Through OpenAPI tool, it says,
Validation status before changes introduced in this branch/PR
Not mandatory to be fixed with this PR, these are the results of the validation before your changes,

please refer to the following link:
https://portal.azure-devex-tools.com/app/branch/bgsky/azure-rest-api-specs/dev-cost-management-Microsoft.CostManagement-2019-01-01...master

**Comment by jhendrixMSFT** [comment]
@AutorestCI regenerate azure-sdk-for-go

File: specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-01-01/costmanagement.json [hunk 1]
```diff
@@ -831,9 +831,11 @@
           "enum": [
             "WeekToDate",
             "MonthToDate",
+            "BillingMonthToDate",
```
Comment (sergey-shandar) [inline specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-01-01/costmanagement.json]: @bgsky could you add new enum items to the end?

**Review by sergey-shandar — CHANGES_REQUESTED** [review]

File: specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-01-01/costmanagement.json (hunk 1 above)
Comment (sergey-shandar) [reply specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-01-01/costmanagement.json]: There are some model validation errors. Could you fix them?

Comment (bgsky) [reply specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-01-01/costmanagement.json]: @sergey-shandar We addressed your comments by moving enum values to the end. In terms of model validation errors. A separate PR fixed them recently, and there is no error from new iteration. Please review and we expect to merge fast.

**Review by sergey-shandar — APPROVED** [review]

**Pull request merged**